drivers/unix/ip_unix.cpp

```diff
@@ -108,22 +109,31 @@ IPAddress IPUnix::_resolve_hostname(const String &p_hostname, Type p_type) {
 	int s = getaddrinfo(p_hostname.utf8().get_data(), nullptr, &hints, &result);
 	if (s != 0) {
 		ERR_PRINT("getaddrinfo failed! Cannot resolve hostname.");
-		return IPAddress();
+		return;
 	};
 
 	if (result == nullptr || result->ai_addr == nullptr) {
 		ERR_PRINT("Invalid response from getaddrinfo");
 		if (result) {
 			freeaddrinfo(result);
 		}
-		return IPAddress();
+		return;
 	};
 
-	IPAddress ip = _sockaddr2ip(result->ai_addr);
+	struct addrinfo *next = result;
 
-	freeaddrinfo(result);
+	do {
+		if (next->ai_addr == NULL) {
+			next = next->ai_next;
+			continue;
+		}
+		IPAddress ip = _sockaddr2ip(next->ai_addr);
+		if (!r_addresses.find(ip))
+			r_addresses.push_back(ip);
```

Comment:
```suggestion
		if (!r_addresses.find(ip)) {
			r_addresses.push_back(ip);
		}
```

Reply:
This is my normal coding standard too - always use braces.  Change noted